irohad/main/peers_file_reader.hpp

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef IROHA_PEERS_FILE_READER_HPP
+#define IROHA_PEERS_FILE_READER_HPP
+
+#include <boost/optional.hpp>
+#include "interfaces/common_objects/common_objects_factory.hpp"
+#include "interfaces/common_objects/peer.hpp"
+
+namespace iroha {
+  namespace main {
```

Comment:
Why is the single namespace "iroha" not sufficient?

Reply:
According to Python's zen: `Namespaces are one honking great idea -- let's do more of those!`